lib/main/util/robot.ts

```diff
@@ -0,0 +1,83 @@
+import cmd from 'node-cmd'
+import { desktopCapturer } from 'electron'
+import psList from 'ps-list'
+import robot from 'robotjs'
+import Store from 'electron-store'
+
+const store = new Store()
+
+class Robot {
+  async trigger(id: string) {
```

Comment:
Add a comment "Returns whether or not the trigger passed its conditions and performed its sequence."